src/Microsoft.VisualStudio.Threading.Tests/AsyncReaderWriterLockTests.cs

```diff
@@ -611,12 +611,11 @@ public async Task IsPassiveUpgradeableReadLockHeldReturnsTrueForIncompatibleSync
         public async Task IsPassiveWriteLockHeldReturnsTrueForIncompatibleSyncContexts()
         {
             var dispatcher = SingleThreadedTestSynchronizationContext.New();
-            using (var releaser = await this.asyncLock.WriteLockAsync())
```

Comment:
❔ Do we still have tests for the previous use cases?

1. Calling `Dispose` without `ReleaseAsync`
2. Explicitly calling `ReleaseAsync` then `Dispose`
3. Explicitly calling `ReleaseAsync` then `DisposeAsync` (by switching to `await using` but forgetting to remove `ReleaseAsync`

Reply:
Yes, there are a bunch of tests I *didn't* update because they were doing interesting things like that.